db/migrate/20230830115531_remove_timeline_events_with_missing_target.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveTimelineEventsWithMissingTarget < ActiveRecord::Migration[6.1]
+  def up
+    te_missing_targets = TimelineEvent.includes(:target).where('targets.id IS NULL').references(:target)
```

Comment:
```suggestion
    te_missing_targets = TimelineEvent.where.missing(:target)
```

Docs: https://apidock.com/rails/v6.1.3.1/ActiveRecord/QueryMethods/WhereChain/missing